### ISSUE TITLE
Check failed: cudnnSetTensorNdDescriptor error while using Keras

### ISSUE THREAD

**Issue by moonchildz** [keras-team/keras#14934]
**System information**.
- Win10
- TensorFlow installed from pip3
- TensorFlow version 2.2:
- Python version: 3.6.8
- Tesla V100 / 32GB VRAM:

I am meeting an error which I have never seen in my life while using keras.

My model gets 256 x 256 x 256 x 1 np array as an input and My code is so long to write full of it so I write some part of it. input_num = 16 bs=3

     input_3d = M.Input( (None,None,None,1))
     result = on.estimate_voxel_3d(input_3d)

     data_train_s  = od.load_org_3d(route = route_org,shuffle=False, split = 'train'  ,imgnum=input_num,batch_size=bs)
     data_val_s    = od.load_org_3d(route = route_org,shuffle=False, split = 'val'    ,imgnum=input_num,batch_size=bs)
     ognet = M.Model(inputs=input_3d, outputs=result)
     ognet.fit_generator(generator=data_train_s,validation_data=data_val_s,epochs=100,shuffle=True,callbacks=[learnschedule,m_check])

The Sequence code for loading dataset is as follow : 

        def __data_generation(self, file_list_temp):
        ib = []
        nb = []
        count = 0
        #rseq = rnd.sample(range(1,130), self.imgnum)
        for iff, fl in enumerate(file_list_temp):  # 0~3

            voxel_file = self.read_voxel(self.route_main + fl + '/' + 'voxel_' + fl + '.xyz')
            confocal_list = os.listdir(self.route_main + fl + '/img/')
            conf_len = len(confocal_list)
            rseq_norm = self.sample_img(self.imgnum)
            rseq_middle = rseq_norm - 0.5
            i_accum = []
            vox_img = np.zeros((256,256,256))
            for ir, r in enumerate(rseq_norm):
                r_int = int(r * conf_len) + 1
                r_int2 = int(rseq_middle[ir]*conf_len)+128
                if r_int2<0 or r_int2>255: print('Overflow Err')
                #print(r_int2)
                ii = cv2.imread(self.route_main+fl+'/img/'+str(r_int)+'.jpg',0)/255.0 #1024x1024
               # r_array = np.full((1024, 1024, 1), r)
                i = np.expand_dims(ii,-1)  #1024 x 1024 x 1
                i_rsz = cv2.resize(i,dsize=None, fx=0.25, fy=0.25)  #256 x 256 x 1
                vox_img[:,:,r_int2] = i_rsz
            ib.append( np.expand_dims(vox_img,-1))
            nb.append(voxel_file)
            count += 1
        #print('data shape:',np.array(ib).shape,np.array(nb).shape)
        return np.array(ib), np.array(nb)

Here I checked the shape of input (256,256,256,1) and output ( 64,64,64) is output correctly.
And the network looks like : 

    def estimate_voxel_3d(input):
    x1 = L.Conv3D(filters=61, kernel_size=(5, 5, 5), strides=1, padding='same')(input)  # 256x 256 x 256 x 64
    x1 = L.BatchNormalization()(x1)
    x1 = L.LeakyReLU(0.1)(x1)
    x1 = L.Conv3D(filters=64, kernel_size=(3, 3, 3), strides=1, padding='same')(x1)  # 256
    x1 = L.BatchNormalization()(x1)
    x1 = L.LeakyReLU(0.1)(x1)
    x1 = L.Conv3D(filters=64, kernel_size=(3, 3, 3), strides=1, padding='same')(x1)  # 256
    x1 = L.BatchNormalization()(x1)
    x1 = L.LeakyReLU(0.1)(x1)

    x2 = L.Conv3D(filters=128, kernel_size=(3, 3, 3), strides=2, padding='same')(x1)  # 128x 128 x 128 x 128
    x2 = L.BatchNormalization()(x2)
    x2 = L.LeakyReLU(0.1)(x2)
    x2 = L.Conv3D(filters=128, kernel_size=(3, 3, 3), strides=1, padding='same')(x2)  # 128
    x2 = L.BatchNormalization()(x2)
    x2 = L.LeakyReLU(0.1)(x2)
    x2 = L.Conv3D(filters=128, kernel_size=(3, 3, 3), strides=1, padding='same')(x2)  # 128
    x2 = L.BatchNormalization()(x2)
    x2 = L.LeakyReLU(0.1)(x2)

    x3 = L.Conv3D(filters=256, kernel_size=(3, 3, 3), strides=2, padding='same')(x2)  # 64 x 64 x 64 x 256
    x3 = L.BatchNormalization()(x3)
    x3 = L.LeakyReLU(0.1)(x3)
    x3 = L.Conv3D(filters=256, kernel_size=(3, 3, 3), strides=1, padding='same')(x3)  # 64
    x3 = L.BatchNormalization()(x3)
    x3 = L.LeakyReLU(0.1)(x3)
    x3 = L.Conv3D(filters=256, kernel_size=(3, 3, 3), strides=1, padding='same')(x3)  # 64
    x3 = L.BatchNormalization()(x3)
    x3 = L.LeakyReLU(0.1)(x3)

    x4 = L.Conv3D(filters=512, kernel_size=(3, 3, 3), strides=2, padding='same')(x3)  # 32 x 32 x 32 x 512
    x4 = L.BatchNormalization()(x4)
    x4 = L.LeakyReLU(0.1)(x4)
    x4 = L.Conv3D(filters=512, kernel_size=(3, 3, 3), strides=1, padding='same')(x4)  # 32
    x4 = L.BatchNormalization()(x4)
    x4 = L.LeakyReLU(0.1)(x4)
    x4 = L.Conv3D(filters=512, kernel_size=(3, 3, 3), strides=1, padding='same')(x4)  # 32
    x4 = L.BatchNormalization()(x4)
    x4 = L.LeakyReLU(0.1)(x4)

    x3_ = L.UpSampling3D(size=(2,2,2))(x4)
    x3_ = L.Concatenate(-1)([x3_,x3])
    x3_ = L.Conv3D(filters=256, kernel_size=(3, 3, 3), strides=1, padding='same')(x3_)  # 64 x 64 x 64 x 256
    x3_ = L.BatchNormalization()(x3_)
    x3_ = L.LeakyReLU(0.1)(x3_)
    x3_ = L.Conv3D(filters=256, kernel_size=(3, 3, 3), strides=1, padding='same')(x3_)  # 64
    x3_ = L.BatchNormalization()(x3_)
    x3_ = L.LeakyReLU(0.1)(x3_)
    x3_ = L.Conv3D(filters=256, kernel_size=(3, 3, 3), strides=1, padding='same')(x3_)  # 64
    x3_ = L.BatchNormalization()(x3_)
    x3_ = L.LeakyReLU(0.1)(x3_)

    xf = L.Conv3D(filters=64, kernel_size=(3, 3, 3), strides=1, padding='same')(x3_)    # 64 x 64 x 64 x 64
    xf = L.BatchNormalization()(xf)
    xf = L.LeakyReLU(0.1)(xf)
    xf = L.Conv3D(filters=1, kernel_size=(3, 3, 3), strides=1, padding='same')(x3_) # 64 x 64 x 64 x 1
    xf = L.BatchNormalization()(xf)
    xf = L.LeakyReLU(0.1)(xf)
    def sq(x):
        squeeze = B.squeeze(x,-1)
        return squeeze
    fc_f = L.Lambda(sq)(xf) # 64 x 64 x 64 (Squeezed )
    return fc_f

I don't think there is any error but don't know why the error occurs like :

    2021-07-13 11:13:20.535468: F tensorflow/stream_executor/cuda/cuda_dnn.cc:516] Check failed: cudnnSetTensorNdDescriptor(handle_.get(), elem_type, nd, dims.data(), strides.data()) == CUDNN_STATUS_SUCCESS (9 vs. 0)batch_descriptor: {count: 3 feature_map_count: 61 spatial: 256 256 256  value_min: 0.000000 value_max: 0.000000 layout: BatchDepthYX}

I check that the size of input is (3,256,256,256,1) and output is (3,64,64,64), And checking that feature_map_count(61) is same with the filters of first layer. Why this happens ? please anybody tell me and thank you very much.

**Comment by ymodak** [comment]
https://github.com/keras-team/keras/issues/11434#issuecomment-882358614

**Comment by ymodak** [comment]
Closing due to lack of activity. Feel free to reopen if the issue still persists. Thanks!
